src/ControlSystem/ControlErrors/Size/Info.hpp

```diff
@@ -16,6 +16,17 @@ namespace control_system::size {
 
 /// Holds information that is saved between calls of SizeControl.
 struct Info {
+  Info() = default;
+  Info(const Info& rhs);
+  Info& operator=(const Info& rhs);
+  Info(Info&& rhs);
+  Info& operator=(Info&& rhs);
+
+  Info(const std::unique_ptr<State>& in_state, double in_damping_time,
```

Comment:
Why take the pointer by reference instead of value?  That would match the current interface.